mscore/pagesettings.h

```diff
@@ -22,29 +22,42 @@
 #define __PAGESETTINGS_H__
 
 #include "ui_pagesettings.h"
+#include "libmscore/mscore.h"
+#include "libmscore/page.h"
 
 namespace Ms {
 
 class MasterScore;
 class Score;
 class Navigator;
 
+// For converting from Ms::Units to a two-char string used as a suffix
+static const char* unitSuffixes[] = {"px", "mm", "in", "sp"};
```

Comment:
I basically said the same as to where it belongs, also to fix that compiler warning this causes. As to translations, I don't think we should translate them, these are units, and as such international?
But then again we do translate mm, sp and inch elsewhere (I've just checked to be sure), so indeed should here too. Guess a bunch of `Q_TRANSLATE_NOOP()` should do here?